edx_reverification_block/xblock/static/js/checkpoint_edit.js

```diff
@@ -17,7 +17,7 @@ function CheckpointEditBlock(runtime, element) {
     function validate(data) {
         var errors = [];
         if ( !validators.string(data.related_assessment) ) {
-            errors.push(gettext("Related Assessment field cannot be empty."));
+            errors.push(gettext("The Associated Assessment field cannot be empty."));
         }
         if ( !validators.number(data.attempts) ) {
             errors.push(gettext("Attempts field should be a positive number."));
```

Comment:
I didn't see this error message when I was working in the sandbox earlier so didn't suggest a revision then. Let's go with this:

You must select a value other than 0 for the Verification Attempts field.

*(Is there a reason this field doesn't default to 1?)*